tooling/inspector/src/main.rs

```diff
@@ -0,0 +1,77 @@
+use std::{path::PathBuf, process::exit};
+
+use acir::circuit::ExpressionWidth;
+use clap::Parser;
+use noirc_artifacts::program::ProgramArtifact;
+use noirc_artifacts_info::{count_opcodes_and_gates_in_program, show_info_report, InfoReport};
+
+#[derive(Debug, Clone, Parser)]
+pub(crate) struct Args {
```

Comment:
Can you make this have be a `noir-inspector info ./target/program.json` command rather than just `noir-inspector ./target/program.json` so we can more easily add extra commands in future?

Reply:
Done!